jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/io/FrameFlusher.java

```diff
@@ -293,10 +295,10 @@ public String toString()
     private final int bufferSize;
     private final Generator generator;
     private final int maxGather;
-    private final Object lock = new Object();
+    private final Locker lock = new Locker();
```

Comment:
I'm using a Locker... not for any real reason other than we use it elsewhere. Synchronized is probably identical